docs/troubleshooting-guide.md

```diff
@@ -1,112 +1,255 @@
+```
+https://YourStorageAccount.blob.core.windows.net/inputs?sv=2018-03-28&sr=c&si=inputs-keysig=somestring
+```
+
+After the change, **sr=c&si=inputs-key** should be the order in your SAS URL. <br/>
+
+Update all the SAS URLs similarly and retry your workflow.
+
+### All TES tasks for my workflow are done running, but the trigger JSON file is still in the "inprogress" directory in the workflows container
+The root cause is most likely memory pressure on the host Linux VM because [blobfuse](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-how-to-mount-container-linux#overview) processes grow to consume all physical memory. 
+
+You may see the following Cromwell container logs as a symptom:
+> Cromwell shutting down because it cannot access the database):
+Shutting down cromid-5bd1d24 as at least 15 minutes of heartbeat write errors have occurred between 2020-02-18T22:03:01.110Z and 2020-02-18T22:19:01.111Z (16.000016666666667 minutes
+
+OR
+> Failed to instantiate Cromwell System. Shutting down Cromwell.
+liquibase.exception.LockException: Could not acquire change log lock.  Currently locked by 012ec19c3285 (172.18.0.4) since 2/19/20 4:10 PM
+
+To mitigate, please resize your VM in the resource group to a machine with at least 14GB memory/RAM. Any workflows still in progress will not be affected.
+
+![Resize VM](/docs/screenshots/resize-VM.png)
+
+
+## Setup
+### Setup Cromwell on Azure for multiple users in the same Azure subscription
+This section is COMING SOON.
+
+### Debug my Cromwell on Azure installation that ran into an error
+When the Cromwell on Azure installer is run, if there are errors, the logs are printed in the terminal. Most errors are related to insufficient permissions to create resources in Azure on your behalf, or intermittent Azure failures. In case of an error, we terminate the installation process and begin deleting all the resources in the Resource Group if already created. <br/>
+
+Deleting all the resources in the Resource Group may take a while but as soon as you see logs that the Batch account was deleted, you may exit the current process using Ctrl+C or Command+C on terminal/command prompt/PowerShell. The deletion of other Azure resources can continue in the background on Azure. Re-run the installer after fixing any user errors like permissions from the previous try.
+
+If you see an issue that is unrelated to your permissions, and re-trying the installer does not fix it, please file a bug on our GitHub issues.
+
+### Upgrade my Cromwell on Azure instance
+Starting in version 1.x, for convenience, some configuration files are hosted on your Cromwell on Azure Storage account, in the "configuration" container - `containers-to-mount`, and `cromwell-application.conf`. You can modify and save these file using Azure Portal UI "Edit Blob" option or simply upload a new file to replace the existing one. Please create the "configuration" container in the if it isn't there already.<br/>
```

Comment:
"Please create the "configuration" container in the if it isn't there already" - incomplete?